--- glucose/core/Solver.cc	2024-08-11 00:52:43.490483402 +0800
+++ Solver.cc	2024-08-11 01:06:38.710493099 +0800
@@ -1616,7 +1616,7 @@
                 decisions++;
                 next = pickBranchLit();
                 if(next == lit_Undef) {
-                    printf("c last restart ## conflicts  :  %d %d \n", conflictC, decisionLevel());
+                    // printf("c last restart ## conflicts  :  %d %d \n", conflictC, decisionLevel());
                     // Model found:
                     return l_True;
                 }
